doc/install/python.rst

```diff
@@ -84,7 +91,7 @@ The following optional flags can be used to configure the installation:
   See :ref:`install-architecture` for details.
 * ``--arch``: CPU micro-architecture to target. The advised default is ``native``.
   See `here <https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html>`_ for a full list of options.
-* ``--makejobs``: Specify the amount of jobs to ``make`` the project with for faster build times on multicore systems. By default set to ``2``.
+* ``-j``: Specify the amount of concurrent jobs for building the project with for faster build times on multicore systems. By default set to ``2``.
```

Comment:
This and other flags: they now need to be passed at CMake args, right? We need to rewrite this part of the docs a bit.

Reply:
No, not this one. It's the one that goes to sk-build ... but the rest.